react/week4/src/components/UsersFetch.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { UserProvider } from './UserContext';
+import UsersSearch from './UsersSearch';
+import UsersList from './UsersList';
+
+export default function UsersFetch() {
+    const [users, setUsers] = useState("")
```

Comment:
Isn't this a search word? So the name should be user (as singular) or searchText or query or...